alembic/versions/4893d060429b_performance_statistics.py

```diff
@@ -0,0 +1,33 @@
+"""performance statistics
+
+Revision ID: 4893d060429b
+Revises: 7b5f75fc2c08
+Create Date: 2019-03-16 15:40:25.595553
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4893d060429b'
+down_revision = '7b5f75fc2c08'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB, ENUM
+import datetime
+
+
+def upgrade():
+    op.create_table(
+        'performance_stats',
+        sa.Column('id', sa.Integer, primary_key=True),
+        sa.Column('project_id', sa.Integer, sa.ForeignKey('project.id', ondelete='CASCADE'), nullable=False),
```

Comment:
indexing stats on project_id and/or other frequently filtered fields can be added to improve db performance

Reply:
I thought about it and decided to not add anything since at least at the beginning this table is going to be fairly small, not sure if you had anything specific in mind for some particular reason.